modules/virt-add-disk-to-vm.adoc

```diff
@@ -24,22 +26,34 @@ Use this procedure to add a virtual disk to a {object}.
 
 .Procedure
 
-. From the *{object-gui}* tab, select your {object}.
-. Select the *Disks* tab.
-. Click *Add Disks* to open the *Add Disk* window.
-. In the *Add Disk* window, specify *Source*, *Name*, *Size*, *Interface*, and *Storage Class*.
+. Click *Workloads* -> *Virtualization* from the side menu.
+. Click the *{object-gui}s* tab.
+. Select a {object} to open the *{object-gui} Overview* screen.
```

Comment:
We say "screen" a lot in our docs, but the ISG says to only use "screen" if you're referring to the physical surface of a display. I think the instances of "screen" in this PR could all be changed to "page" but please correct me if I am wrong about that.

Reply:
Yes, we do say "screen" a lot in our docs in spite of what the ISG says. Those three steps appear in all web console procedures, not just in this PR. I think changing "screen" to "page" would involve a larger effort and a separate PR.